CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Optimizely Python SDK Changelog
 
+## 3.8.1
+May 28th, 2021
+
+### New Features
```

Comment:
If this "new features", the tag should be 3.9.0.
If this is "bug fix", then keep 3.8.1